env.yaml

```diff
@@ -11,3 +11,5 @@ dependencies:
   - pip:
     - pre-commit
     - ipywidgets
+    - kaleido
+    - git+https://github.com/cytomining/pycytominer@c90438fd7c11ad8b1689c21db16dab1a5280de6c
```

Comment:
Consider updating to the latest Pycytominer SHA here.

Reply:
updated hash from newest commit